test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -96,6 +98,27 @@ private static void testMemoryLimit(String valueToSet, String expectedTraceValue
             .shouldMatch("Memory Limit is:.*" + expectedTraceValue);
     }
 
+    // JDK-8292083
+    // Ensure that Java ignores container memory limit values above the host's physical memory.
+    private static void testContainerMemExceedsPhysical()
+            throws Exception {
+
+        Common.logNewTestCase("container memory limit exceeds physical memory");
+
+        DockerRunOptions opts = Common.newOpts(imageName);
+
+        // first run: establish physical memory in test environment and derive
+        // a bad value one power of ten larger
+        String goodMem = Common.run(opts).firstMatch("total physical memory: (\\d+)", 1);
```

Comment:
`goodMem` will return `null` if there is no match. Could you please handle this and bail out the test with some meaningful message. Right now this fails the test run on unpatched with `--memory null0` being an illegal value when passed to the container engine.

Reply:
Thanks, resolved in 4984ddd9c9f68c00aaa8935769e9ac925690c2fb.

Reply:
I've added an assert in 4984ddd9c9f to resolve this. Thanks!